tests/foreman/api/test_repository.py

```diff
@@ -1944,6 +1946,91 @@ def test_positive_sync_publish_promote_cv(self):
         self.assertGreaterEqual(len(result.stdout), 1)
 
 
+class SRPMRepositoryIgnoreContentTestCase(APITestCase):
+    """Test whether SRPM content can be ignored during sync.
+
+    In particular sync of duplicate SRPMs would fail when using the flag
+    ``ignorable_content``.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a product and an org which can be re-used in tests."""
+        super().setUpClass()
+        cls.org = entities.Organization().create()
```

Comment:
Does it change anything? I think API doesn't create entities for last created org by default, so if you are not creating your stuff in this organization explicitly, your test is just spending time on creating org that will never be used.

Reply:
Re-tested. Not required. Removed.